core/app/assets/javascripts/refinery/admin.js.erb

```diff
@@ -577,13 +577,17 @@ var page_options = {
   },
 
   page_part_dialog: function(){
+    var slugOptions = {
+      separator: '_'
```

Comment:
Yeah this is for programmatical reference, nothing shown anywhere. I agree with @bricesanchez